tests/users/test_dao_social_sign_in.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+
+from app.api.dao.user import UserDAO
+from tests.base_test_case import BaseTestCase
+from app.database.models.user import UserModel
+
+
+class TestSocialSignIn(BaseTestCase):
+    def test_create_user_using_social_sign_in(self):
+        user_data = dict(
+            id_token="test_token",
+            name="test_name",
+            email="test_email"
+        )
+        social_sign_in_type = "test_type"
+
+        # Call create user using social sign in method
+        UserDAO.create_user_using_social_login(user_data, social_sign_in_type)
+
+        # Test the user created
+        user = UserDAO.get_user_by_email("test_email")
+        self.assertTrue(user.name == user_data["name"])
+        self.assertTrue(user.username is None)
+        self.assertTrue(user.password_hash is None)
+
+        # Test the social sign in details of the user created
+        social_sign_in_details = UserDAO.get_social_sign_in_details(user.id, "test_type")
+        self.assertTrue(social_sign_in_details.id_token == user_data["id_token"])
```

Comment:
why not use `assertEqual` in that way you avoid doing the comparison with ==
Can you do that for the other lines

Reply:
Done